### PR TITLE
Cherry-picks for 0.6.2 release

### PR DESCRIPTION
**Pull request by XVilka**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Bugfixes and documentation updates

**Test plan**

CI is green

### CODE REVIEW

**Comment by XVilka** [comment]
Looks like only missing things are remaining `?v` invocations @ret2libc

**Comment by p371jah** [comment]
Maybe these changes too #3752

**Comment by XVilka** [comment]
@pelijah I already imported them.

**Comment by XVilka** [comment]
@pelijah sorry, I squashed them, didn't notice the title

**Comment by p371jah** [comment]
No it is older one fix pack. You imported #3779 which is newer. I don't see commits from #3752 in stable.

File: librz/include/rz_util/rz_strbuf.h [hunk 1]
```diff
@@ -12,7 +12,6 @@ typedef struct {
 	size_t len; // string length in chars or binary buffer size
 	char *ptr; // ptr replacing buf in case strlen > sizeof(buf)
 	size_t ptrlen; // string length + 1 or binary buffer size
-	bool weakref; // ptr is not owned
```
Comment (ret2libc) [inline librz/include/rz_util/rz_strbuf.h]: @XVilka this can't be done in a patch release, in theory. It should be done in the next release.

Comment (XVilka) [reply librz/include/rz_util/rz_strbuf.h]: Good point, will remove that commit

**Review by wargio — APPROVED** [review]

**Comment by XVilka** [comment]
Something is really wrong with NetBSD (happens also on `dev`):
```
+ python3.8 -m ensurepip --user Traceback (most recent call last):   File "<string>", line 6, in <module>   File "/usr/pkg/lib/python3.8/runpy.py", line 207, in run_module     return _run_module_code(code, init_globals, run_name, mod_spec)   File "/usr/pkg/lib/python3.8/runpy.py", line 97, in _run_module_code     _run_code(code, mod_globals, init_globals,   File "/usr/pkg/lib/python3.8/runpy.py", line 87, in _run_code     exec(code, run_globals)   File "/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl/pip/__main__.py", line 31, in <module>   File "/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl/pip/_internal/cli/main.py", line 68, in main   File "/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl/pip/_internal/commands/__init__.py", line 114, in create_command   File "/usr/pkg/lib/python3.8/importlib/__init__.py", line 127, in import_module     return _bootstrap._gcd_import(name[level:], package, level)   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import   File "<frozen importlib._bootstrap>", line 991, in _find_and_load   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked   File "<frozen importlib._bootstrap>", line 655, in _load_unlocked   File "<frozen importlib._bootstrap>", line 618, in _load_backward_compatible   File "<frozen zipimport>", line 259, in load_module   File "/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl/pip/_internal/commands/install.py", line 16, in <module>   File "<frozen zipimport>", line 259, in load_module   File "/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl/pip/_internal/cli/req_command.py", line 21, in <module>   File "<frozen zipimport>", line 259, in load_module   File "/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl/pip/_internal/index/package_finder.py", line 30, in <module>   File "<frozen zipimport>", line 259, in load_module   File "/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl/pip/_internal/req/__init__.py", line 8, in <module>   File "<frozen zipimport>", line 259, in load_module   File "/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl/pip/_internal/req/req_install.py", line 44, in <module>   File "<frozen zipimport>", line 259, in load_module   File "/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl/pip/_internal/operations/install/wheel.py", line 39, in <module>   File "<frozen zipimport>", line 259, in load_module   File "/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl/pip/_vendor/distlib/scripts.py", line 16, in <module>   File "<frozen zipimport>", line 259, in load_module   File "/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl/pip/_vendor/distlib/compat.py", line 83, in <module>   File "/usr/pkg/lib/python3.8/xmlrpc/client.py", line 138, in <module>     from xml.parsers import expat   File "/usr/pkg/lib/python3.8/xml/parsers/expat.py", line 4, in <module>     from pyexpat import * ModuleNotFoundError: No module named 'pyexpat' Traceback (most recent call last):   File "/usr/pkg/lib/python3.8/runpy.py", line 194, in _run_module_as_main     return _run_code(code, main_globals, None,   File "/usr/pkg/lib/python3.8/runpy.py", line 87, in _run_code     exec(code, run_globals)   File "/usr/pkg/lib/python3.8/ensurepip/__main__.py", line 5, in <module>     sys.exit(ensurepip._main())   File "/usr/pkg/lib/python3.8/ensurepip/__init__.py", line 209, in _main     return _bootstrap(   File "/usr/pkg/lib/python3.8/ensurepip/__init__.py", line 128, in _bootstrap     return _run_pip(args + [p[0] for p in _PROJECTS], additional_paths)   File "/usr/pkg/lib/python3.8/ensurepip/__init__.py", line 37, in _run_pip     return subprocess.run(cmd, check=True).returncode   File "/usr/pkg/lib/python3.8/subprocess.py", line 516, in run     raise CalledProcessError(retcode, process.args, subprocess.CalledProcessError: Command '['/usr/pkg/bin/python3.8', '-c', '\nimport runpy\nimport sys\nsys.path = [\'/tmp/tmpcfwb944t/setuptools-56.0.0-py3-none-any.whl\', \'/tmp/tmpcfwb944t/pip-23.0.1-py3-none-any.whl\'] + sys.path\nsys.argv[1:] = [\'install\', \'--no-cache-dir\', \'--no-index\', \'--find-links\', \'/tmp/tmpcfwb944t\', \'--user\', \'setuptools\', \'pip\']\nrunpy.run_module("pip", run_name="__main__", alter_sys=True)\n']' returned non-zero exit status 1.
```
The crux of the problem seems to be missing `pyexpat` module for some reason: `ModuleNotFoundError: No module named 'pyexpat'`

**Pull request merged**